systemtest/src/main/java/io/strimzi/systemtest/utils/kafkaUtils/KafkaRebalanceUtils.java

```diff
@@ -45,7 +47,14 @@ private static Condition rebalanceStateCondition(String resourceName) {
 
     public static void waitForKafkaRebalanceCustomResourceState(String resourceName, KafkaRebalanceState state) {
         KafkaRebalance kafkaRebalance = KafkaRebalanceResource.kafkaRebalanceClient().inNamespace(kubeClient().getNamespace()).withName(resourceName).get();
-        ResourceManager.waitForResourceStatus(KafkaRebalanceResource.kafkaRebalanceClient(), kafkaRebalance, state.toString());
+        ResourceManager.waitForResourceStatus(KafkaRebalanceResource.kafkaRebalanceClient(), kafkaRebalance, state.toString(), ResourceOperation.getTimeoutForKafkaRebalanceState(state));
+    }
+
+    public static String annotateKafkaRebalanceResource(String resourceName, KafkaRebalanceAnnotation annotation) {
+        LOGGER.info("Annotating KafkaRebalance:{} with annotation {}", resourceName, annotation.toString());
+        return ResourceManager.cmdKubeClient().namespace(kubeClient().getNamespace())
+            .execInCurrentNamespace("annotate", "kafkarebalance", resourceName, "strimzi.io/rebalance=" + annotation.toString())
```

Comment:
```suggestion
            .execInCurrentNamespace("annotate", "kafkarebalance", resourceName, KafkaRebalanceAssemblyOperator.ANNO_STRIMZI_IO_REBALANCE + "=" + annotation.toString())
```

Reply:
Yeah that looks better!

Reply:
why not ... 

```java
LOGGER.info("Annotating KafkaRebalance:{} with annotation {}={}", resourceName, annotation.toString());
        return ResourceManager.cmdKubeClient().namespace(kubeClient().getNamespace())
            .execInCurrentNamespace("annotate", "kafkarebalance", resourceName, KafkaRebalanceAssemblyOperator.ANNO_STRIMZI_IO_REBALANCE, annotation.toString())
```

avoiding the last string concatenation :P

Reply:
but won't that miss out the "="?

Reply:
I have put it in the formatting string "annotation {}={}"

Reply:
Here is needed that `=`, how @samuel-hawker and @tomncooper said.
```
io.strimzi.test.k8s.exceptions.KubeClusterException: `oc --namespace cruise-control-isolated-test annotate kafkarebalance my-cluster strimzi.io/rebalance approve` got status code 1 and stderr:
------
error: at least one annotation update is required
```

Reply:
I mislead with the previous logger instruction :)